src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -325,6 +355,7 @@ private static boolean isAnonInnerClassInit(DetailAST lcurly) {
     // -@cs[JavaNCSS] getDetails() method is a huge SWITCH, it has to be monolithic
     // -@cs[ExecutableStatementCount] getDetails() method is a huge SWITCH, it has to be monolithic
     // -@cs[NPathComplexity] getDetails() method is a huge SWITCH, it has to be monolithic
+    // -@cs[CyclomaticComplexity] getDetails() method is a huge SWITCH, it has to be monolithic
```

Comment:
What about such suppression ?
```
-@cs[JavaNCSS,ExecutableStatementCount,*Complexity] method is a huge SWITCH, it has to be monolithic
```
if description is too long - we could wrap it on second line.